.circleci/config.yml

```diff
@@ -55,10 +54,6 @@ jobs:
     <<: *node-base
     docker:
       - image: circleci/node:6
-  node-v8:
```

Comment:
If you're removing this I think there's no point in keeping `node-v4` and `node-v6` above.